synapse/tests/test_model_inet.py

```diff
@@ -1729,3 +1730,74 @@ def test_model_inet_rfc2822_addr(self):
             self.eq(n2[0], n3[0])
             self.eq(n2[0], n4[0])
             self.eq(n1[0], n5[0])
+
+    def test_model_inet_http(self):
+
+        with self.getRamCore() as core:
+
+            tick = 0x01010101
+            flow = s_common.guid()
+            host = s_common.guid()
+            body = s_common.guid()
+
+            props = {
+                'flow': flow,
+                'time': tick,
+                'host': host,
+                'method': 'HeHe',
+                'path': '/foo/bar',
+                'query': 'baz=faz&woot=haha',
+                'body': body,
+            }
+
+            requ = core.formTufoByProp('inet:http:request', '*', **props)
+
+            iden = requ[1].get('inet:http:request')
+
+            self.eq(requ[1]['inet:http:request:flow'], flow)
+            self.eq(requ[1]['inet:http:request:time'], tick)
+            self.eq(requ[1]['inet:http:request:host'], host)
+            self.eq(requ[1]['inet:http:request:method'], 'HeHe')
+            self.eq(requ[1]['inet:http:request:path'], '/foo/bar')
+            self.eq(requ[1]['inet:http:request:query'], 'baz=faz&woot=haha')
+
+            node = core.formTufoByProp('inet:http:reqhead', (iden, ('User-Agent', 'BillyBob')))
```

Comment:
For demonstrative purposes, it would be good to show how the ``Host:`` parameter would be encoded